tmt/steps/prepare/install.py

```diff
@@ -389,14 +398,17 @@ def sort_packages(self) -> None:
         self.required_packages = []
         for package in self.repository_packages:
             try:
-                output = self.guest.execute(Command('rpm', '-q', package), silent=True)
-                assert output.stdout
-                self.debug(f"Package '{output.stdout.strip()}' already installed.")
+                self.rpm_check(package=package)
             except tmt.utils.RunError:
-                if self.skip_missing:
-                    self.recommended_packages.append(package)
-                else:
-                    self.required_packages.append(package)
+                try:
+                    # Dependencies can also be files, let's check those as well
```

Comment:
What about choosing the right check based on the require name? E.g. use directly `-qf` if it starts with a `/`?